src/SwedbankPay.Sdk.Infrastructure/PayeeInfoResponse.cs

```diff
@@ -0,0 +1,46 @@
+﻿using SwedbankPay.Sdk.PaymentOrders;
+using System;
+
+namespace SwedbankPay.Sdk
+{
+    internal class PayeeInfoResponse : Identifiable, IPayeeInfo
+    {
+        public PayeeInfoResponse(Uri id) : base(id)
+        {
+        }

```

Comment:
Isn't this constructor superfluous? `PayeeInfoResponse` is always created from a DTO containing an `Id` that can be passed to `Identifiable.ctor()`, no?

```suggestion
```

Reply:
You are correct.